### PR TITLE
Minor UI fixes

### PR DESCRIPTION
**Pull request by gz**

### CODE REVIEW

File: web-ui/src/streaming/PipelineTable.tsx [hunk 1]
```diff
@@ -389,7 +383,11 @@ export default function PipelineTable() {
 
   const startPipelineClick = useCallback(
     (curRow: PipelineDescr) => {
-      if (!pipelineActionLoading && isLaunching.get(curRow.pipeline_id) != ClientPipelineStatus.PAUSED) {
+      if (
+        !pipelineActionLoading &&
+        (isLaunching.get(curRow.pipeline_id) != ClientPipelineStatus.PAUSED ||
```
Comment (mihaibudiu) [inline web-ui/src/streaming/PipelineTable.tsx:388]: [ClientPipelineStatus.PASSED, ...].includes(isLaunching.get(...)) evaluates the get only once.

File: web-ui/src/streaming/introspection/IntrospectionTable.tsx [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+import { PipelineDescr, ProgramDescr, PipelineStatus, OpenAPI } from 'src/types/manager'
+import { parse } from 'csv-parse'
+import { parseProjectSchema } from 'src/types/program'
+
+export type IntrospectionTableProps = {
+  pipelineDescr: PipelineDescr | undefined
+  name: string | undefined
+}
+
+export const IntrospectionTable = ({ pipelineDescr, name }: IntrospectionTableProps) => {
```
Comment (mihaibudiu) [inline web-ui/src/streaming/introspection/IntrospectionTable.tsx:16]: why "Introspection"?
Why not "browse"?

File: web-ui/src/streaming/introspection/IntrospectionTable.tsx [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+    enabled: pipelineDescr !== undefined && pipelineDescr.program_id !== undefined
+  })
+  useEffect(() => {
+    if (!projectQuery.isLoading && !projectQuery.isError && name) {
+      if (projectQuery.data && projectQuery.data.schema) {
+        const program = parseProjectSchema(projectQuery.data)
+        const view = program.schema['outputs'].find(v => v.name === name)
+
+        if (view) {
+          const id = [{ field: 'genId', headerName: 'genId' }]
```
Comment (mihaibudiu) [inline web-ui/src/streaming/introspection/IntrospectionTable.tsx:31]: what's this for?

**Review by mihaibudiu — APPROVED** [review]

File: web-ui/src/streaming/introspection/IntrospectionTable.tsx (hunk 2 above)
Comment (gz) [reply web-ui/src/streaming/introspection/IntrospectionTable.tsx:16]: Ok will change in next PR when I'm using leonids new APIs

File: web-ui/src/streaming/introspection/IntrospectionTable.tsx (hunk 3 above)
Comment (gz) [reply web-ui/src/streaming/introspection/IntrospectionTable.tsx:31]: it's an identifier for the row so I know what row to update when a change comes in

File: web-ui/src/streaming/PipelineTable.tsx (hunk 1 above)
Comment (gz) [reply web-ui/src/streaming/PipelineTable.tsx:388]: Ok will change in next PR when I'm using leonids new APIs

**Pull request merged**